<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_TQCS1</class>
 <widget class="QWidget" name="Dlg_TQCS1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox_Func_Enab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Syn_Function_Enable</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>471</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Synchronization Subnetworks Style Setting</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Manual Select</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>206</width>
       <height>55</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QRadioButton" name="radioButton_SYN_Style">
        <property name="text">
         <string>Synchronous Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_ASYN_Style">
        <property name="text">
         <string>Asynchronous Networks</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QRadioButton" name="radioButton_Routine_Auto_Select">
       <property name="text">
        <string>Routine Auto Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Manual_Select">
       <property name="text">
        <string>Manual Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Dead Volt Select</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QCheckBox" name="checkBox_L1_Dead_Volt">
       <property name="text">
        <string>L1 Dead Volt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_L2_Dead_Volt">
       <property name="text">
        <string>L2 Dead Volt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_ALL_Dead_Volt">
       <property name="text">
        <string>All Dead Volt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>U Phase Select</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QRadioButton" name="radioButton_Phase_A">
       <property name="text">
        <string>Phase A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Phase_B">
       <property name="text">
        <string>Phase B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Phase_C">
       <property name="text">
        <string>Phase C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Apply">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Fetch">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>Fetch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_Default">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Tr.U1-U2 (0- 360 °)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>fmax (45-55Hz)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Dfptmax"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>°</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Udead(2- 50 V)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_TrU1U2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Udead"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_TsynDuration"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_fmax"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>fmin (45-55Hz)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>T-syn.Duration （2-30s）</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Hz/s</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_fmin"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>dΔf/dtmax （0.1-2Hz/s）</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_Synchron"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>T Synchron (0-60s)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>d(f1-f2)/dt （0.1-2Hz/s）</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_Synchron2"/>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_68">
      <property name="text">
       <string>Hz/s</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>s</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>s</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_D_Umax2"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_Dfmax"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_43">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_80">
      <property name="text">
       <string>Umax (20-120V)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Δαmax (0- 50 °)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_D_fmax2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>ΔUasync (2- 50 V)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>ΔUsync (2- 50 V)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_46">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Δfasync (0.1-1 Hz)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>°</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_DUmax"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_TCB"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Damax"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_Umin"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>T_CB(30-2000 ms)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_44">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_48">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_45">
      <property name="text">
       <string>mHz</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Δfsync(10-60mHz)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_Umax"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>Umin (20- 120V)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
